logic/round.js

```diff
@@ -233,5 +253,37 @@ Round.prototype.land = function () {
 		}.bind(this));
 };
 
+/**
+ * Calls:
+ * - updateVotes
+ * - updateMissedBlocks
+ * - flushRound
+ * - applyRound
+ * - updateVotes
+ * - flushRound
+ * - restoreRoundSnapshot
+ * - restoreVotesSnapshot
+ * @implements {updateVotes}
+ * @implements {updateMissedBlocks}
+ * @implements {flushRound}
+ * @implements {applyRound}
+ * @implements {restoreRoundSnapshot}
+ * @implements {restoreVotesSnapshot}
+ * @return {function} call result
+ */
+Round.prototype.backwardLand = function () {
+	return this.updateVotes()
+		.then(this.updateMissedBlocks.bind(this))
+		.then(this.flushRound.bind(this))
+		.then(this.applyRound.bind(this))
+		.then(this.updateVotes.bind(this))
```

Comment:
Yes its redundant but this changes the least amount of behavior by preserving the original construction.